1-fizzbuzz/src/main.rs

```diff
@@ -0,0 +1,20 @@
+fn fizzbuzz(sz: usize) -> String {
+    let div_3 = sz % 3 == 0;
+    let div_5 = sz % 5 == 0;
+
+    if div_3 && div_5 {
```

Comment:
Looks good :+1: 
When it comes to idiomatic style, Rust works well with expression-driven control flow similar to Haskell - you can write it more subtly as
```rust
if div_3 && div_5 {
  String::from("fizzbuzz")
} else if div_3 {
  String::from("fizz")
} else if div_5 {
  String::from("buzz")
} else {
  format!("{sz}")
}
```
because it appears as last expression in block, and `if` is actually expression too - it behaves similarly to blocks, but requires branches to agree on final type (which I should've mentioned, sorry).